src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -20,21 +19,26 @@
 public class Climber extends SubsystemBase {
 
   private CANSparkMax m_leftClimberMotor, m_rightClimberMotor;
-  private AnalogInput m_pot;
+  private CANEncoder m_leftEncoder, m_rightEncoder;
   private DoubleSolenoid m_climberSolenoid;
 
 
   public Climber() {
     m_leftClimberMotor = new CANSparkMax(RobotMap.CAN.CLIMBER_LEFT_SPARKMAX, MotorType.kBrushless);
     m_rightClimberMotor = new CANSparkMax(RobotMap.CAN.CLIMBER_RIGHT_SPARKMAX, MotorType.kBrushless);
     m_climberSolenoid = new DoubleSolenoid(RobotMap.PCM.CLIMBER_A, RobotMap.PCM.CLIMBER_B);
-    m_pot = new AnalogInput(RobotMap.AIO.CLIMBER_POT);
+
+    m_leftEncoder = m_leftClimberMotor.getEncoder();
+    m_rightEncoder = m_rightClimberMotor.getEncoder();
```

Comment:
Nitpick: Instead of creating these encoder objects, let's just use `getEncoder()` everywhere.